.travis.yml

```diff
@@ -44,8 +44,17 @@ matrix:
       python: "3.6"
       env: TEST_UNIT=1 TEST_COVER=1
     - os: linux
-      python: "3.7-dev"
+      dist: xenial
+      sudo: true
+      python: "3.7"
       env: TEST_UNIT=1
+    - os: linux
+      dist: xenial
+      sudo: true
+      python: "3.8-dev"
+      env: TEST_UNIT=1
+  allow_failures:
+    - python: "3.8-dev"
```

Comment:
Perhaps a small comment why we accept failures from this env? Does it glitch?

Reply:
I think it passes; I just prefer keeping dev pythons under `allow_failures` as a matter of course, because the feature set isn't stable and if it *were* to fail, we shouldn't lose any sleep over it until it actually gets released. I can add a comment to this effect.

Reply:
Thanks, yeah a comment like this in the file would help me if I forget the reason in a few months :)